Swift Vocabulary/Swift Vocabulary/DefinitionViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  DefinitionViewController.swift
+//  Swift Vocabulary
+//
+//  Created by Bryan Cress on 5/12/20.
+//  Copyright © 2020 Bryan Cress. All rights reserved.
+//
+
+import UIKit
+
+class DefinitionViewController: UIViewController {
+    
+    var vocabWord: VocabularyWord?
+    
+    //MARK: Properties
+    
+    @IBOutlet weak var nameLabel: UILabel!
+    
+    
+    @IBOutlet weak var nameTextView: UITextView!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        updateViews()
+    }
+        func updateViews() {
+            if let vocabWord = vocabWord {
+                nameLabel.text = vocabWord.word
+                nameTextView.text = vocabWord.definition
+            }
+        }
+        // Do any additional setup after loading the view.
+    }
+    
+
+    /*
+    // MARK: - Navigation
```

Comment:
It's okay to delete these comments. If it's not going to be used or needed later on